.template/variants/web/Dangerfile.rb

```diff
@@ -5,5 +5,8 @@
 
     # Runs ESLint on modified and added files in the PR
     eslint.lint
+
+    # Runs Stylelint on modified and added files
```

Comment:
as `stylelint.lint` lints all files, should we update the comment? What do you think?
```suggestion
    # Runs Stylelint on all files
```

Reply:
Updated in https://github.com/nimblehq/rails-templates/pull/329/commits/054e458c14a319f39360cf979e3e28ef43569a22!